packages/ui5-task-transpile/lib/transpile.js

```diff
@@ -15,16 +15,28 @@ const log = require("@ui5/logger").getLogger("builder:customtask:transpile");
  * @returns {Promise<undefined>} Promise resolving with undefined once data has been written
  */
 module.exports = function({workspace, dependencies, options}) {
+    const plugins = []
+      .concat(
+        options.configuration && options.configuration.removeConsoleStatements
+          ? "transform-remove-console"
```

Comment:
IMO this needs to be an Array `["transform-remove-console"]`

Reply:
Adding the array here has no effect though I'm not sure if this true for all JS versions. I can add it to be sure.

Reply:
Nevermind, I see what you mean. I will fix this.